plugins/woocommerce/src/Admin/Features/ShippingPartnerSuggestions/DefaultShippingPartners.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\ShippingPartnerSuggestions;
+
+/**
+ * Default Shipping Partners
+ */
+class DefaultShippingPartners {
+
+	/**
+	 * Get default specs.
+	 *
+	 * @return array Default specs.
+	 */
+	public static function get_all() {
+		$asset_base_url      = WC()->plugin_url() . 'assets/images/shipping_partners/';
```

Comment:
I think there's a missing slash between `woocommerce` and `assets` in the path - I get a 404 with the URL that's generated but its found if i add the slash
<img width="822" alt="image" src="https://user-images.githubusercontent.com/27843274/225818356-9aa7d8e9-9d20-41bc-beab-4e8fa9bfc868.png">